lib/manager/fvm/extract.ts

```diff
@@ -0,0 +1,39 @@
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { logger } from '../../logger';
+import type { PackageDependency, PackageFile } from '../types';
+
+interface FvmConfig {
+  flutterSdkVersion: string;
+}
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
+): PackageFile | null {
+  let fvmConfig: FvmConfig;
+  try {
+    fvmConfig = JSON.parse(content);
+  } catch (err) {
+    logger.debug({ packageFile }, 'Invalid FVM config');
```

Comment:
```suggestion
    logger.debug({ packageFile, err }, 'Invalid FVM config');
```
So you can see the probably parsing error in logs.